.github/workflows/net-6-pipeline-canary.yaml

```diff
@@ -0,0 +1,44 @@
+name: NET 6 Canary
+
+on:
+  workflow_dispatch:
+  schedule:
+  # Run this once a day to check if everything is still working as expected.
+    - cron: "0 0 * * *"
+
+jobs:
+
+  build-deploy:
+      runs-on: ubuntu-latest
+      steps:
+        - uses: actions/checkout@v2  
+        - name: Build and Deploy
+          uses: ./.github/actions/sam-build-and-deploy
+          with:
+            aws-access-key: ${{ secrets.AWS_ACCESS_KEY_ID }}
+            aws-secret-key: ${{ secrets.AWS_SECRET_ACCESS_KEY }}
+            aws-region: ${{ secrets.AWS_REGION }}
+            dotnet-version: '6.0.x'
+            template-file-path: ./src/NET6/template.yaml
+            stack-name: net-6-base
```

Comment:
The canaries use the same stack name as the workflows that update the performance results for the GitHub README. We could separate them out if we want, but I'm not sure it's necessary.